git-cpp-check

```diff
@@ -68,7 +68,8 @@ for filename, ranges in changed_lines.iteritems():
   cpplint.ProcessFile(filename, 0)
   sys.stderr = save_stderr
   for linenum, message in _cpplint_file_errors:
-    if any((linenum in r for r in ranges)):
+    # Sometimes cpplint reports linenum=0 instead of the acutal line.
+    if not linenum or any((linenum in range(r.start, r.start + r.count) for r in ranges)):
```

Comment:
Please change to explicit 'if linenum == 0 or ...' and put this as a second alternative, so:
if any(..) or linenum == 0: